bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ValidationResult.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace ElectionGuard
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct ValidationResult
```

Comment:
this struct is a mirror of the one in the c++ code but it is not yet hooked up so they are not equivalent - they simply have the same structure right now.